magprime/templates/magprime_reports/volunteer_food.html

```diff
@@ -22,7 +22,8 @@
         <thead>
             <tr>
                 <th>Volunteer</th>
-                <th>Sandwich Preferences</th>
+	        <th>Badge Number</th>
+	        <th>Sandwich Preferences</th>
```

Comment:
This has tabs instead of spaces, but @timrxd had to leave for awhile, so I'll have him fix the tabs/spaces issue as part of a separate PR.